### PR TITLE
Enable MemberImportVisibility check on all targets

### PR DESCRIPTION
**Pull request by rnro**
Enable MemberImportVisibility check on all targets. Use a standard string header and footer to bracket the new block for ease of updating in the future with scripts.

### CODE REVIEW

**Comment by simonjbeaumont** [comment]
Is this expected to be passing? 🤔

**Review by simonjbeaumont — APPROVED** [review]

File: Package.swift [hunk 1]
```diff
@@ -23,30 +23,24 @@ let swiftSettings: [SwiftSetting] = [
...
-            name: "OpenAPIRuntime",
-            targets: ["OpenAPIRuntime"]
-        )
-    ],
-    dependencies: [
-        .package(url: "https://github.com/apple/swift-http-types", from: "1.0.0"),
-    ],
+    platforms: [.macOS(.v10_15), .macCatalyst(.v13), .iOS(.v13), .tvOS(.v13), .watchOS(.v6), .visionOS(.v1)],
+    products: [.library(name: "OpenAPIRuntime", targets: ["OpenAPIRuntime"])],
+    dependencies: [.package(url: "https://github.com/apple/swift-http-types", from: "1.0.0")],
```
Comment (czechboy0) [inline Package.swift:28]: Curious why this got reformatted now, even though the CI was green previously as well?

File: Package.swift [hunk 2]
```diff
@@ -23,30 +23,24 @@ let swiftSettings: [SwiftSetting] = [
...
-            name: "OpenAPIRuntimeTests",
-            dependencies: ["OpenAPIRuntime"],
+            dependencies: [.product(name: "HTTPTypes", package: "swift-http-types")],
             swiftSettings: swiftSettings
-        ),
+        ), .testTarget(name: "OpenAPIRuntimeTests", dependencies: ["OpenAPIRuntime"], swiftSettings: swiftSettings),
     ]
 )
+
+// ---    STANDARD CROSS-REPO SETTINGS DO NOT EDIT   --- //
```
Comment (czechboy0) [inline Package.swift:38]: Are we missing a START: here?

File: Package.swift (hunk 1 above)
Comment (czechboy0) [reply Package.swift:28]: Yeah, the other repos didn't get reformatted, so I wonder if we could undo it here: https://github.com/swift-server/swift-openapi-async-http-client/pull/43/files

Comment (rnro) [reply Package.swift]: Sorry, that was accidental - I've reverted the formatting changes

File: Package.swift (hunk 2 above)
Comment (rnro) [reply Package.swift:54]: This is to try and have the lengths of the two blocks the same and have the text centered so it isn’t too ugly. We can revisit the exact structure of this the next time the common block is updated

**Pull request merged**